src/scales/timeScale.ts

```diff
@@ -12,6 +12,7 @@ export module Scales {
     constructor() {
       super();
       this._d3Scale = d3.time.scale();
+      this.autoDomain();
```

Comment:
To set the domain to padded `_defaultExtent()`. I guess we could test for it? It'd be kind of tautological though.